test/run/ch-fromhost.bats

```diff
@@ -261,6 +261,19 @@ fromhost_ls () {
     [[ $status -eq 1 ]]
     [[ $output = *'duplicate image'* ]]
     fromhost_clean_p "$img"
+
+    # ldconfig gives no shared library path (#324)
+    #
+    # (I don't think this is the best way to get ldconfig to fail, but I
+    # couldn't come up with anything better. E.g., bad ld.so.conf or broken
+    # .so's seem to produce only warnings.)
```

Comment:
Another mode would be to break ch-run (that's the case I ran into).

With this PR, I get a useful error message that would give me a good start on figuring out what was wrong.

```
$ ch-fromhost --nvidia ex2/rootfs
ch-run[73]: error: Permission denied (charliecloud.c:508 13)
ch-fromhost: empty path from ldconfig
```